lib/services/screen-builder-service.ts

```diff
@@ -60,35 +59,51 @@ export class ScreenBuilderService implements IScreenBuilderService {
 		}).future<void>()();
 	}
 
-	private promptGenerate(generatorName: string, type?: string): IFuture<void> {
+	private promptGenerate(generatorName: string, screenBuilderOptions?: IScreenBuilderOptions): IFuture<void> {
+		var scaffolderData = this.createScaffolder(generatorName, screenBuilderOptions);
+		var scaffolder = scaffolderData.scaffolder;
+		var type = screenBuilderOptions.type || ScreenBuilderService.DEFAULT_SCREENBUILDER_TYPE;
+
+		if(type === ScreenBuilderService.DEFAULT_SCREENBUILDER_TYPE) {
+			scaffolder.promptGenerate(type, screenBuilderOptions.answers, scaffolderData.callback);
```

Comment:
aah, correct, I didn't notice it is not our promptGenerate method, sorry :smile: